src/data.js

```diff
@@ -1,8 +1,8 @@
 const productData = [
   {
     images: [
-      "https://www.fillmurray.com/200/200",
-      "https://www.fillmurray.com/201/201"
+      "https://www.fillmurray.com/1200/1200",
+      "https://www.fillmurray.com/1201/1201"
```

Comment:
```suggestion
      "https://www.fillmurray.com/1200/1200",
      "https://www.fillmurray.com/1202/1202"
```
change to 1202 instead